app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/BusinessName.scala.html

```diff
@@ -20,35 +20,32 @@
 @import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.config.AppConfig
 
 @import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.models.BusinessNameModel
-@import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.templates.main_template
-@import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.helpers._
+@import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.templates.PrincipalMainTemplate
+@import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.templates.injected.TextInputHelper
+@import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.templates.injected.ContinueButton
 
-@(businessNameForm: Form[BusinessNameModel], postAction: Call, isEditMode: Boolean, backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-
-@inputContent = {
-    <p>@messages("business.name.line_1")</p>
-}
+@this(
+    mainTemplate: PrincipalMainTemplate,
+    form: FormWithCSRF,
+    textInputHelper: TextInputHelper,
+    continueButton: ContinueButton
+)
 
-@main_template(title = Messages("business.name.title"), form = Some(businessNameForm)) {
+@(businessNameForm: Form[BusinessNameModel], postAction: Call, isEditMode: Boolean, backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-    @back_link(backUrl)
 
-    @summaryErrorHelper(businessNameForm)
+@mainTemplate(title = messages("business.name.title"), optForm = Some(businessNameForm), backLink = Some(backUrl)) {
 
     @form(action = postAction) {
 
-        @inputHelper(
+        @textInputHelper(
             field = businessNameForm(businessName),
             label = messages("business.name.heading"),
-            content = Some(inputContent),
-            parentForm = businessNameForm
+            hint = Some(messages("business.name.line_1",
```

Comment:
```suggestion
            hint = Some(messages("business.name.line_1")),
```
I think you removed these two brackets by accident